File: Data Structures & Algorithms/concatenation-of-array/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number[]}
     */
    getConcatenation(nums) {
        if(!nums.length) return [];
        let n = nums.length;
        let res = [n*2];
        for(let i=0; i<n; i++){
            res[i] = nums[i];
            res[n+i] = nums[i];
        }
        return res;
    }
}

File: Data Structures & Algorithms/missing-number/submission-0.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int ans = 0;
        for(int i=0; i<=n; i++){
            ans = ans+i;
        }
        int res = 0;
        for(int i=0; i<nums.length; i++){
            res = res+nums[i];
        }

        return ans - res;
    }
}
